src/main/java/fruit/shop/records/parser/RecordsSaver.java

```diff
@@ -0,0 +1,8 @@
+package fruit.shop.records.parser;
+
+import java.util.List;
+import java.util.Map;
+
+public interface RecordsSaver {

```

Comment:
why are they `Savers` in `parser` package. Parsers are a more accurate description